Actors/ActorCounter/Scripts/upgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+azure servicefabric application package copy CounterActorApplication fabric:ImageStore > /dev/null 2>&1
```

Comment:
so the script should be fired from root level.. we should capture that in the readme if not already done